mos-platform/nes-cnrom/bank.c

```diff
@@ -6,12 +6,32 @@
 #include <rompoke.h>
 #include "bank.h"
 
-__attribute__((section(".zp.bss"))) char _BANK_SHADOW;
```

Comment:
This should definitely be called `_BANK_NEXT`, since it entirely controls the bank as of next NMI, and it has no relation to the current bank state.

Reply:
Done.